shoop/admin/modules/products/views/forms.py

```diff
@@ -263,3 +284,23 @@ def save(self, commit=True):
 class ProductImageMediaFormSet(ProductMediaFormSet):
     allowed_media_kinds = [ProductMediaKind.IMAGE]
     form_class = ProductImageMediaForm
+
+    def save(self, commit=True):
+        """
+        Add first saved image as  primary image for this product by default if none is selected as such
```

Comment:
The docstring is misleading, since it does not have a title and therefore reader gets the idea that the "adding first saved image..." is the only thing that is being done, which is not true.

Add a docstring title and wrap it to 79 chars/line (as our style guide sais about docstring formatting).

e.g.
```python
def save(self, commit=True):
    """
    Save the form.

    In addition add the first saved image as primary image for the
    product if none is selected as such.
    """
```